Examples/restcomm-olympus-nopod/restcomm-olympus/Utils.m

```diff
@@ -25,6 +25,23 @@
 #import "RCUtilities.h"
 
 
+//keys
+NSString *const kContactKey = @"contacts";
```

Comment:
Good idea on introducing constants for these keys 👍 . Actually, at some point we need to introduce them within the SDK to make it easier to use keys for the parameters argument. But let's leave that for later